rules/dispatch.go

```diff
@@ -169,8 +170,8 @@ func (r *DefaultRuleRenderer) hostDispatchChains(
 			"",
 			ChainDispatchFromHostEndpoint,
 			"",
-			fromEndRules,
-			toEndRules,
+			fromEndForwardRules,
+			toEndForwardRules,
```

Comment:
I think this specific change is wrong, because we use `HostFromEndpointPfx` in `HostEndpointToMangleChains`, for pre-DNAT policy, not `HostFromEndpointForwardPfx`.  But the rest of this commit looks spot on.  (I missed that a single HostDispatchChains call is responsible for creating the dispatch chains for both normal and AOF policy.)

Reply:
Thanks for spotting this!